Dangerfile

```diff
@@ -1,3 +1,12 @@
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet.
+has_wip_label = github.pr_labels.any? { |label| label.include? "WIP" }
+has_wip_title = github.pr_title.include? "[WIP]" || github.pr_title.include? "WIP"
+
+if has_wip_label || has_wip_title
+  message("Skipping Danger since PR is classed as Work in Progress")
```

Comment:
Small thing, how about `Skipping Danger since PR state is in Work In Progress state`

Reply:
No, in your sentence, `state` word is duplicated 😄  Any other idea? 🤔

Reply:
Oh, how about `Skipping Danger since PR state is in Work In Progress (WIP)`